src/domain/alerts/alerts.repository.ts

```diff
@@ -38,46 +39,97 @@ export class AlertsRepository implements IAlertsRepository {
   }
 
   async handleAlertLog(chainId: string, log: AlertLog): Promise<void> {
+    const emails = await this.emailRepository.getVerifiedEmailsBySafeAddress({
+      chainId,
+      // TODO: This is the address of the module, _not_ the Safe
+      // Discussion in https://github.com/safe-global/safe-client-gateway/pull/923
+      safeAddress: log.address,
+    });
+
+    if (emails.length === 0) {
+      return this.loggingService.debug(
```

Comment:
nitpick: Logging for me it's more of a side-effect of the function. While it works because of the matching return types, if that changes in the future we might have issues with the return type on this function.

Reply:
Addressed in 47123ceec383947a3757e50dde2e110255be40a7.